daphne/src/vdaf/mod.rs

```diff
@@ -325,6 +326,21 @@ pub enum VdafPrepState {
     Prio3Field128(Prio3PrepareState<Field128, 16>),
 }
 
+#[cfg(any(test, feature = "test-utils"))]
+impl deepsize::DeepSizeOf for VdafPrepState {
+    fn deep_size_of_children(&self, _context: &mut deepsize::Context) -> usize {
+        // This method is, as documented, an estimation of the size of the children. Since it can't
+        // be known for this type due to it's encapsulation, I will count the size of it as 0.
+        //
+        // TODO: Make a better estimation.
```

Comment:
We would need to know (1) which VDAF is in use and (2) whether we're the Leader or the Helper. How does `deepsize::Context` work? Could we use it to plumb what we need here?

For what it's worth, for Prio2 and prio3 the size is constant, as we're using the `Share::Helper` variant in all cases.

Reply:
The size being constant for the helper means this function is correct for the helper case, as this estimation is supposed to only account for heap allocations/stuff behind pointers.

`deepsize::Context` is an opaque type, it only has functionality for the `deepsize` crate, we can't use it for anything